packages/website/src/clientModules.js

```diff
@@ -1,3 +1,3 @@
-if (typeof window !== 'undefined' && typeof window.document !== 'undefined') {
```

Comment:
Ha, my instinct here was that back in the day you'd find a lot of SSR renderers would polyfill in `window` for compatibility. Very annoying. Glad that's no longer a necessary check!

Reply:
No, just `typeof window` is sufficient in all cases😏